model/src/w3updtmd.F90

```diff
@@ -1648,7 +1723,24 @@ SUBROUTINE W3UIC5( FLFRST )
 !/
       END SUBROUTINE W3UIC5
 !/ ------------------------------------------------------------------- /      
-      
+!
+!> @author H. L. Tolman @date 28-Mar-2014
+!> @param[inout] VA Spectra in 1-D or 2-D representation (points to same address).
+!> @returns VA Spectra in 1-D or 2-D representation (points to same address).
+!> @brief Update ice map in the wave model.
+!> 
+!> @details
+!> @parblock
```

Comment:
Don't use the parblock tag. Instead, just leave a blank line between two paragraphs, as you are already doing. So:

```
!> The first sentence is the brief description.
!>
!> Now we have another paragraph of information.
!>
!> Now we have yet another paragraph.

```

Reply:
That's good to know, I had thought the parblock was necessary.  I'll make this change.

Reply:
All parblocks have been removed.